extensions/resteasy-reactive/quarkus-rest-client-microprofile/deployment/src/test/java/io/quarkus/resteasy/reactive/client/timeout/Resource.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.resteasy.reactive.client.timeout;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import io.vertx.core.Vertx;
+
+@Path("/")
+@Produces(MediaType.TEXT_PLAIN)
+@Consumes(MediaType.TEXT_PLAIN)
+public class Resource {
+
+    @Inject
+    Vertx vertx;
+
+    @Path("/slow")
+    @GET
+    public String slow() throws InterruptedException {
+        Thread.sleep(5000L);
+        CompletableFuture<String> result = new CompletableFuture<>();
```

Comment:
What's the deal with this?

Reply:
a leftover :(

Reply:
At least it's honest and upfront.

Reply:
😆

Reply:
I meant `CompletableFuture` is a leftover. The`sleep` is to test timeouts